lib/diagnostic.rb

```diff
@@ -1,11 +1,11 @@
-# Ruby enumerable assessment
-class Assessment
+# Ruby enumerable diagnostic
+class Diagnostic
   # Question 1
   # Write ruby code to return the longest entry in the
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.length!
```

Comment:
Sorry, this isn't right. We want to find the element with the longest length, not the size of the collection.